ui/base/clipboard/clipboard_android.cc

```diff
@@ -102,14 +102,6 @@ void ClipboardMap::Set(const std::string& format, const std::string& data) {
 
     Java_Clipboard_setText(env, clipboard_manager_.obj(), str.obj());
   } else if (format == kHTMLFormat) {
-    // Android's API for storing HTML content on the clipboard requires a plain-
-    // text representation to be available as well. ScopedClipboardWriter has a
-    // stable order for setting clipboard data, ensuring that plain-text data
-    // is available first. Do not write to the clipboard when only HTML data is
-    // available, because otherwise others apps may not be able to paste it.
-    if (!ContainsKey(map_, kPlainTextFormat))
-      return;
-
```

Comment:
Neither disable or enable HTML P&P unconditionally does not make any sense . Can we expose this key via `XWalkPreferences`, so that developer can control it via this key.